### PR TITLE
Toon shader UI rework

### PR DESCRIPTION
**Pull request by DMotorygin**
JIRA TICKET: RPRMAYA-3245

PURPOSE: Change of Toon Shader UI was requested by TMS

EFFECT FOR THE USER: several parameters (9) of Toon Shader that were set by sliders are now set by ramp. Necessary features for such UI were added.

### CODE REVIEW

File: FireRender.Maya.Src/scripts/AERPRToonMaterialTemplate.mel [hunk 1]
```diff
@@ -10,6 +10,230 @@
...
+			-cw 1 123 -annotation (localizedAttrName("Selected Value")) $scName`;
+		}
+		
+		string $lmax;
+		if ( $adaptiveScaling ) {
+			$lmax = `text -label "1.0" ($rampName+"LX")`;
+		}
+		string $rframe = `frameLayout -lv 0 -cll 0 ($rampForm + "fr")`;
+		string $widgetName = `gradientControl
+								-at $nodeAttr
```
Comment (Vsevolod1983) [inline FireRender.Maya.Src/scripts/AERPRToonMaterialTemplate.mel:72]: a bit weird code alignment

Comment (Vsevolod1983) [reply FireRender.Maya.Src/scripts/AERPRToonMaterialTemplate.mel:72]: its a mix of spaces and tabs all along the file

**Review by Vsevolod1983 — CHANGES_REQUESTED** [review]
pls fix tabs vs spaces problem in the mel file

**Review by Vsevolod1983 — APPROVED** [review]

**Comment by DavidSpeicher** [comment]
QA approved

**Pull request merged**